### PR TITLE
Add net461 framework version

### PR DESCRIPTION
**Pull request by IanCMarshall**
Fix for https://github.com/Azure/azure-libraries-for-net/issues/282

### CODE REVIEW

File: src/ResourceManagement/ResourceManager/Microsoft.Azure.Management.ResourceManager.Fluent.csproj [hunk 1]
```diff
@@ -5,18 +5,17 @@
 
   <PropertyGroup>
     <Description>Provides resource group and resource management (Fluent) capabilities for Microsoft Azure.</Description>
-    <Version>1.13.0</Version>
+    <Version>1.14.0</Version>
```
Comment (hovsepm) [inline src/ResourceManagement/ResourceManager/Microsoft.Azure.Management.ResourceManager.Fluent.csproj]: Please revert the version upgrade - we do that during release process

File: src/ResourceManagement/ResourceManager/Authentication/AzureCredentials.cs [hunk 2]
```diff
@@ -146,20 +153,13 @@ public async override Task ProcessHttpRequestAsync(HttpRequestMessage request, C
...
-#if !NETSTANDARD
+#if (!NETSTANDARD && !NET461)
                 else if (userLoginInformation != null)
                 {
                     credentialsCache[adSettings.TokenAudience] = await UserTokenProvider.LoginSilentAsync(
                         userLoginInformation.ClientId, TenantId, userLoginInformation.UserName,
                         userLoginInformation.Password, adSettings, TokenCache.DefaultShared);
                 }
-#endif
-#if NETSTANDARD
```
Comment (hovsepm) [inline src/ResourceManagement/ResourceManager/Authentication/AzureCredentials.cs]: LoginByDeviceCodeAsync should be supported in .NetCore. Any particular reason to remove this functionality?

File: src/ResourceManagement/ResourceManager/Microsoft.Azure.Management.ResourceManager.Fluent.csproj [hunk 3]
```diff
@@ -5,18 +5,17 @@
...
   </PropertyGroup>
-  
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <TargetFrameworks>net452;net461;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.Authentication" Version="2.3.2" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.Authentication" Version="2.3.3" />
```
Comment (hovsepm) [inline src/ResourceManagement/ResourceManager/Microsoft.Azure.Management.ResourceManager.Fluent.csproj]: Please also upgrade the Auth package version in `Tests\AzSdk.test.reference.props` file.

File: src/ResourceManagement/ResourceManager/Authentication/AzureCredentials.cs (hunk 2 above)
Comment (IanCMarshall) [reply src/ResourceManagement/ResourceManager/Authentication/AzureCredentials.cs]: The entire UserTokenProvider has been removed in package 2.3.3 for netstandard1.4

**Comment by hovsepm** [comment]
@IanCMarshall I'm closing this PR in favor of https://github.com/Azure/azure-libraries-for-net/pull/439 . Simply moved your fixes to a new branch in main repo to be able to add commits.

**Comment by hovsepm** [comment]
@IanCMarshall  Release 1.15.1 is out. Please upgrade your references.

**Pull request closed without merging**